lib/src/Uncertainty/Algorithm/Simulation/SubsetSampling.cxx

```diff
@@ -97,8 +97,8 @@ void SubsetSampling::run()
   if (getMaximumCoefficientOfVariation() != ResourceMap::GetAsNumericalScalar("Simulation-DefaultMaximumCoefficientOfVariation"))
     Log::Warn(OSS() << "The maximum coefficient of variation was set. It won't be used as termination criteria.");
 
-  if (targetProbability_ * N < 1.0)
-    throw InvalidArgumentException(HERE) << "The number of samples per step (" << N << ") should be >= " << ceil(1.0 / targetProbability_);
+  if (conditionalProbability_ * N < 1.0)
+    throw InvalidArgumentException(HERE) << "The number of samples per step (" << N << ") should be >= " << ceil(1.0 / conditionalProbability_);
 
   if (N <= 100)
```

Comment:
no hard-coded constant without a mathematical background. Here 100 is arbitrary.